### DISCUSSION TITLE
Customized <myResource>.resw file name supported in Xaml(xUid) and CPP?

### DISCUSSION THREAD

**Discussion by sallyc3** [microsoft/WindowsAppSDK#1841] [Q&A] [answered]
String resource in Strings/Resource.resw works for .xaml : x:Uid="myString"

String resource in Strings/MyResource.resw does not work for .xaml : xUid="/MyResource/myString".  This customized MyResource.resw resource file name works in UWP application.

**Comment by andrewleader** [comment]
@axelandrejs are you the right person to help with this question?

**Comment by sallyc3** [comment]
Hi,

Yes.  I'm the right person on .resw question.

       Thanks.
-Sally

**Comment by andrewleader** [comment, marked as answer]
OP opened a new issue #1842 for this topic, we'll use that issue instead, marking this as answered.
